msteams-platform/samples/virtual-table-entity-references.md

```diff
@@ -0,0 +1,292 @@
+* Sorting isn't supported.
+* Filtering isn't supported.
+* Server driven pagination is supported, with maximum page size being 400.
+
+### Attributes for Graph Planner Task
+
+| Column  | Dataverse Type | Details |
+|---|---|---|
+| `m365_collaborationrootid` | StringType | Collaboration root ID of the collaboration session record is associated with multiple collaboration sessions. This will be returned as a comma delimited string. Note that this attribute won't be returned when retrieving multiple records. |
+| `m365_activechecklistitemcount` | IntegerType | Number of checklist items with value set to false, representing incomplete items. |
+| `m365_appliedcategories` | StringType | Number of checklist items with value set to false, representing incomplete items. |
+| `m365_appliedcategories` | StringType | The categories to which the task has been applied. This attribute is a JSON encoded string, for example "{ \"category1\": true, \"category6\": true, \"category9\": true }" |
+| `m365_assigneepriority` | StringType | Hint used to order items of this type in a list view. The format is defined as outlined in [using order hints in Planner](/graph/api/resources/planner-order-hint-format). |
+| `m365_assignments` | StringType | The set of assignees, the task is assigned to. This attribute is a JSON encoded string for example "{\" 7be...\": {\"assignedBy\": {\"user\": {\"displayName\", \"email\", \"ID\":\" 7be...\"}, \"group\": null, \"application\": null \"device\": null}" |
+| `m365_bucketid` | StringType | Bucket ID to which the task belongs. The bucket needs to be in the plan that the task is in. It's 28 characters long and case-sensitive. [Format validation](/graph/api/resources/planner-identifiers-disclaimer) is done on the service. |
+| `m365_checklistitemcount` | IntegerType | Number of checklist items that are present on the task. |
+| `m365_completedby` | StringType | Identity of the user that completed the task. This attribute is a JSON encoded string, for example {\"user\": {\"displayName\",\"ID\":\"d55...\"}} |
+| `m365_completeddatetime` | DateTimeType | Read-only. Date and time at which the 'percentComplete' of the task is set to '100'. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z |
+| `m365_conversationthreadid` |StringType | Thread ID of the conversation on the task. This is the ID of the conversation thread object created in the group. |
+| `m365_createdby` | StringType | Identity of the user that created the task. This attribute is a JSON encoded string, for example {\"user\": {\"displayName\",\"ID\":\"d55...\"}} |
+| `m365_createddatetime` | DateTimeType | Read-only. Date and time at which the task is created. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z |
```

Comment:
Please change the term `DateTimeType` to `dateTime` everywhere in the entity reference